### PR TITLE
[Add] Checks description

### PR DESCRIPTION
**Pull request by Kristinita**
For example, I know English is bad. I don't know, for example, why not recommend use «so» in the beginning of the sentence. I had to use Google, that to see English language recommendations. It would be nice, if your module users may quick to see links about write-good checks, not spend a lot of time.

Thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,28 +86,36 @@ write-good *.md --no-passive
...
 
 ### `weasel`
-Checks for "weasel words."
+Checks for "weasel words."  
+See [«“Weasel word” — English Wikipedia article»](https://en.wikipedia.org/wiki/Weasel_word).
 
 ### `adverb`
-Checks for adverbs that can weaken meaning: really, very, extremely, etc.
+Checks for adverbs that can weaken meaning: really, very, extremely, etc.  
+See [«Pretty, fairly, really, very, and quit»](http://www.learnersdictionary.com/qa/pretty-fairly-really-very-and-quite).
```
Comment (Bisaloo) [inline README.md]: quit -> quite

Comment (Kristinita) [inline README.md]: 7f546b2. Thanks.

**Comment by RichardLitt** [comment]
I don't think that this PR belongs in this module. Rather, for each submodule - for instance [passive-voice](https://github.com/btford/passive-voice) - you should link to the that module, and justifications can given there. Having them here isn't as relevant as putting links where the code is. Can you reopen these as smaller PRs in each of the submodules?

**Comment by duereg** [comment]
I've got in and updated the cliche, wordiness, and adverb modules with relevant links.

**Comment by RichardLitt** [comment]
@duereg Looks like you need to push to cliche? Not seeing the changes there.

**Comment by RichardLitt** [comment]
@Kristinita For this PR, can you update the versions for the modules in question, and link them in the main README? For instance:

```md
Checks for passive voice. Uses [passive-voice](https://github.com/btford/passive-voice). 
```

**Comment by RichardLitt** [comment]
Ping @duereg and @Kristinita.

**Comment by Kristinita** [comment]
@RichardLitt, for example, I am user of write-good and get warning/error message of write-good. I want quick to see reasons, why I get error. It would be nice, if user can find all information in README.MD — in the most obvious place — of write-good.

> Rather, for each submodule - for instance passive-voice - you should link to the that module, and justifications can given there. Having them here isn't as relevant as putting links where the code is.

I don't think, that it a best idea.

1. Before you answer, I did not know, that write-good submodules is in another GitHub repositories. Perhaps, users also can not find submodules, that read description of write-good.
2. User should take extra actions, that read description of submodules. I think that developer should not to make extra difficulties for users. I think, the description should be in the most obvious place.

Thanks.

**Comment by RichardLitt** [comment]
> I don't think, that it a best idea.

I understand where you're coming from; you're using write-good out-of-the-box, and you want to know why a particular error is being flagged. The issue, for me, is that the submodules are themselves controversial. For instance, I disagree about the use of `passive-voice` - I think this is a particularly needless rule. Linking in write-good to articles about the use of the passive would open up the documentation here to discussion about submodules, which isn't what I want - I think the documentation should stay where the relevant code is. Instead of this, I think it would be better to appropriately link to the submodules in the README. So:

> Before you answer, I did not know, that write-good submodules is in another GitHub repositories. Perhaps, users also can not find submodules, that read description of write-good.

This is why I suggested we link to the modules themselves. I agree that this is a current failing in our documentation. 

> User should take extra actions, that read description of submodules. I think that developer should not to make extra difficulties for users. I think, the description should be in the most obvious place.

I agree that user's shouldn't have to take extra actions, but I think that the short lines we have describing the modules currently in the README is enough. For instance:

```
#### passive

Checks for passive voice. Uses [passive-voice](https://github.com/btford/passive-voice). 
```

I think that this would be fine.

What you suggest in this PR is linking to random articles of varying quality. I would like a more nuanced discussion in the documentation for each submodule, and I think the best way to do that is to not include justifications here, but to link out to them.

**Comment by Kristinita** [comment]
@RichardLitt , maybe would be better to create a file `Check.md` and add in it descriptions of all checks?

See [**example for MarkdownLinter**](https://github.com/mivok/markdownlint/blob/master/docs/RULES.md).

Thanks.

**Comment by RichardLitt** [comment]
@Kristinita Then the documentation would be neither in the modules where it is relevant, nor in the README. I don't think that is a step forward.

**Comment by RichardLitt** [comment]
Closing this for now. Thank you again, @Kristinita, but I'm not sure we want this change to be added in.

**Comment by Kristinita** [comment]
@RichardLitt , `status:need at least some changes`

### 1. Summary

Since 2017, nothing was done to add rules description.

Of course, possible can do better than me, but I think my changes are better than nothing.

### 2. Submodules

Well, let the description be in the submodules.

But at February 2019 [**no links to submodules**](https://github.com/btford/write-good/blob/313d1fd48183875d8580a8087f0aa8a34ab27e5a/README.md) in README. Users must take the time to find them.

Can you add links to submodules as in [**your example**](https://github.com/btford/write-good/pull/65#issuecomment-288274078)?

```md
#### passive

Checks for passive voice. Uses [**passive-voice**](https://github.com/btford/passive-voice).
```

### 3. Quality

> What you suggest in this PR is linking to random articles of varying quality.

I agree that possible adds more reputable sources.

Thanks.

**Comment by RichardLitt** [comment]
Feel free to open a PR adding links to submodules in the README. :)

**Pull request closed without merging**